config/config/sc-config.yaml

```diff
@@ -1157,17 +1159,19 @@ welcomingDashboard:
 networkPolicies:
   global:
     objectStorageSwift:
-      ips: [] # - set-me-if-enabled
-      ports: [] # - set-me-if-enabled
+      ips:
+        - "set-me-if-enabled"
+      ports:
+        - 5000
```

Comment:
This one can still be templated into `allow-opensearch-master, NetworkPolicy` even when rclone-sync is turned off 
```      - to:
          - ipBlock:
              cidr: 0.0.0.0/0
+         - ipBlock:
+             cidr: set-me-if-enabled
        ports:
          - protocol: TCP
            port: 443
+         - protocol: TCP
+           port: 5000
```
So maybe keep this one as it was.

Reply:
I do not think we use swift for opensearch, so it should not template
I will check

Reply:
should be fixed now